### PR TITLE
Remove SVG fallback

### PR DESCRIPTION
**Pull request by Henni**
(based on https://github.com/owncloud/core/pull/25129)

ToDo:
- [x] remove javascript code for png fallback
- [x] remove png files
- [x] remove `svg` classes in html templates

### CODE REVIEW

**Comment by Henni** [comment]
`/core/img/logo.png` is used inside test files (e.g. `tests/lib/Files/ViewTest.php`)
@DeepDiver1975 @PVince81 can we get rid of these references?

**Comment by DeepDiver1975** [comment]
Please fix the js unit test

````
07:41:08 PhantomJS 2.1.1 (Linux 0.0.0) Core base tests Link functions Images Generates image path with png ext when svg support is not available FAILED
07:41:08 	Expected 'http://localhost/core/img/somefile.svg' to equal 'http://localhost/core/img/somefile.png'.
07:41:08 	/ssd/jenkins/workspace/core-ci-linux-jsunit/database/sqlite/label/SLAVE/core/js/tests/specs/coreSpec.js:247:53
07:41:08 	Expected 'http://localhost:9876/context.html//appsx/testapp/img/somefile.svg' to equal 'http://localhost:9876/context.html//appsx/testapp/img/somefile.png'.
07:41:08 	/ssd/jenkins/workspace/core-ci-linux-jsunit/database/sqlite/label/SLAVE/core/js/tests/specs/coreSpec.js:248:54
07:41:08 ............................
07:41:12 PhantomJS 2.1.1 (Linux 0.0.0) Core base tests SVG extension replacement replaces svg extension with png when SVG not supported FAILED
07:41:12 	Expected '/path/to/myicon.svg?someargs=1' to equal '/path/to/myicon.png?someargs=1'.
07:41:12 	/ssd/jenkins/workspace/core-ci-linux-jsunit/database/sqlite/label/SLAVE/core/js/tests/specs/coreSpec.js:528:13

**Comment by Henni** [comment]
@DeepDiver1975 tests fixed

**Comment by PVince81** [comment]
@DeepDiver1975 another weird JS unit tests failure:
```
09:17:18 + ./autotest-js.sh
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/body-parser/node_modules/raw-body requires bytes@'2.3.0' but will load
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/bytes,
09:17:18 npm WARN unmet dependency which is version 2.1.0
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/body-parser/node_modules/raw-body requires iconv-lite@'0.4.13' but will load
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/body-parser/node_modules/iconv-lite,
09:17:18 npm WARN unmet dependency which is version 0.4.11
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/serve-static/node_modules/send requires depd@'~1.1.0' but will load
09:17:18 npm WARN unmet dependency /ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/node_modules/karma/node_modules/connect/node_modules/depd,
09:17:18 npm WARN unmet dependency which is version 1.0.1
09:17:18 
09:17:18 module.js:340
09:17:18     throw err;
09:17:18           ^
09:17:18 Error: Cannot find module 'zeparser'
09:17:18     at Function.Module._resolveFilename (module.js:338:15)
09:17:18     at Function.Module._load (module.js:280:25)
09:17:18     at Module.require (module.js:364:17)
09:17:18     at require (module.js:380:17)
09:17:18     at Object.<anonymous> (/ssd/jenkins/workspace/core-ci-linux-jsunit@2/database/sqlite/label/SLAVE/build/lib/node_modules/karma/node_modules/socket.io/node_modules/socket.io-client/node_modules/active-x-obfuscator/index.js:1:78)
09:17:18     at Module._compile (module.js:456:26)
09:17:18     at Object.Module._extensions..js (module.js:474:10)
09:17:18     at Module.load (module.js:356:32)
09:17:18     at Function.Module._load (module.js:312:12)
09:17:18     at Module.require (module.js:364:17)
09:17:19 Build step 'Conditional step (single)' marked build as failure
09:17:24 Setting commit status on GitHub for https://github.com/owncloud/core/commit/8ef6f84918ab8ff605f20d227a120f63e689eb76
09:17:25 Finished: FAILURE
```

**Comment by PVince81** [comment]
@Henni I just rebased this onto master, else autotest-js.sh wouldn't run for me

After the rebase all JS tests passed.

File: core/js/js.js [hunk 1]
```diff
@@ -1358,49 +1355,6 @@ if(typeof localStorage !=='undefined' && localStorage !== null){
...
-					if(parts.length===2){
-						var value=parts[1].trim();
-						if(value[0]==='"'){
-							value=value.substr(1,value.length-2);
-						}
-						headers[parts[0].toLowerCase()]=value;
-					}
-				}
-			});
-			if(headers["content-type"]!=='image/svg+xml'){
```
Comment (PVince81) [inline core/js/js.js]: I wonder what the hell this was about...

**Comment by PVince81** [comment]
Code looks good, GUI still looks good :+1:

**Comment by PVince81** [comment]
@owncloud/designers @DeepDiver1975 @owncloud/javascript second review ?

**Comment by jancborchardt** [comment]
Can't check the functionality atm cause laptop is in repair but great initiative @Henni! Finally we can get rid of all those dupe images …

**Comment by MorrisJobke** [comment]
Tested and still works 👍

**Pull request merged**